images/hugo/image.yaml

```diff
@@ -4,6 +4,24 @@ versions:
       extractTagsFrom:
         package: hugo
       subvariants:
+        - suffix: -extended
+          options:
+            - extended
+
         - suffix: -dev
           options:
-            - dev
\ No newline at end of file
+            - dev
+
+        - suffix: -extended-dev
+          options:
+            - dev
+            - extended
+
+options:
+  extended:
+    contents:
+      packages:
+        add:
+          - hugo-extended
+        remove:
+          - hugo
```

Comment:
It looks like the "remove" section is unfortunately no longer supported. We may need separate apko configs for the extended versions since CI fails with:
```
❕ x86_64    | buildImage failed: installing apk packages: unable to install files for pkg hugo-extended: unable to install file over existing one, different contents: usr/bin/hugo
```

Reply:
@jdolitsky did you see my comment above? I think there is a solid case for only shipping the extended variant in images. Happy to go either way though, just let me know.

Reply:
ah i see.. will raise this with the team